lib/std/os/windows/ntdll.zig

```diff
@@ -23,36 +23,95 @@ const FILE_BASIC_INFORMATION = windows.FILE_BASIC_INFORMATION;
 const SIZE_T = windows.SIZE_T;
 const CURDIR = windows.CURDIR;
 
-pub extern "NtDll" fn RtlGetVersion(
+pub extern "ntdll" fn RtlGetVersion(
```

Comment:
I've seen issues where not using `NtDll` in that exact casing causes LLVM/LLD to go into an infinite loop when linking a program. This might still be the case

Reply:
Let me know if you have a source for this, it works on my machine (the best source, obviously) and has always worked with rustc too which also uses LLVM. Maybe it's something on older versions? We updated to LLVM 13 recently.

Reply:
Ironically, the source for it was on my machine. But it was also a few months ago. Let me try to generate a quick demo and will report back if it doesn't happen anymore as it could be fixed by LLVM update like you've said.

Reply:
I remember hitting it with `NtWaitForAlertByThreadId` but it looks like its fixed now.

Reply:
Great to hear! Can't imagine how that happened in the first place, as the file *is* named `ntdll.lib`, all lowercase...